opentelemetry-sdk/src/logs/record.rs

```diff
@@ -37,6 +39,22 @@ pub struct LogRecord {
     pub attributes: Option<Vec<(Key, AnyValue)>>,
 }
 
+impl Default for LogRecord {
```

Comment:
Since we are only overriding one field, it might be better to keep the auto implementation of `Default` trait and instead update the call site like this:

```rust
fn create_log_record(&self) -> Self::LogRecord {
        LogRecord { attributes : Some(Vec::with_capacity(PREALLOCATED_ATTRIBUTE_CAPACITY)), .. Default::default() }
    }
```

Reply:
yes this indeed is the better approach. Have modified accordingly.  Thanks.